instrumentation/opentelemetry-instrumentation-celery/src/opentelemetry/instrumentation/celery/__init__.py

```diff
@@ -277,3 +292,27 @@ def _trace_retry(*args, **kwargs):
         # Use `str(reason)` instead of `reason.message` in case we get
         # something that isn't an `Exception`
         span.set_attribute(_TASK_RETRY_REASON_KEY, str(reason))
+
+    def update_task_start_time(self, task_id):
+        cur_time = default_timer()
+        elapsed_time = (
+            cur_time - self.task_id_to_start_time[task_id]
+            if task_id in self.task_id_to_start_time
+            else cur_time
+        )
+        self.task_id_to_start_time[task_id] = elapsed_time
```

Comment:
I don't follow this logic. Is it updating the start time, or is it calculating the elapsed? Why is the start time marked as elapsed time?

Reply:
It's the start time, I changed the name that it will be clearer